specification/app/resource-manager/readme.md

```diff
@@ -65,11 +65,33 @@ input-file:
   - Microsoft.App/preview/2024-02-02-preview/Subscriptions.json
   - Microsoft.App/preview/2024-02-02-preview/Usages.json
   - Microsoft.App/preview/2024-02-02-preview/FunctionsExtension.json
+  - Microsoft.App/preview/2024-02-02-preview/SessionPools.json
+  - Microsoft.App/preview/2024-02-02-preview/Sessions.json
 directive:
   - suppress: PatchBodyParametersSchema
     from: JavaComponents.json
     reason: |
       Java Component is using componentType as the discriminator. While the discriminator is a required property, this rule prevent it being present in the patch request body.
+  - suppress: LroErrorContent
+    from: SessionPools.json
+    reason: |
+      Using the same error response as other APIs.
+  - suppress: EnumInsteadOfBoolean
+    from: SessionPools.json
+    reason: |
+      Boolean is needed for authEnabled and enableEgress
```

Comment:
>Boolean is needed for authEnabled and enableEgress [](http://example.com/codeflow?start=6&length=50)

Standard guidance is: replace boolean/switch properties with a more meaningful enum whenever possible. 

A boolean will forever have two valid values (true or false). A string enum type is always preferred. Also, properties should always provide better values just than True and False. For example two switches "isTypeA" and "isTypeB" should be replaced with one enum "type": [A, B, DefaultType]. Enums are always a more flexible and future proof option because they allow additional values to be added in the future in a non-breaking way, e.g. [Enabled, Disabled, Suspended, Deallocated]. 

Note: do NOT define a 'boolean enum' with two values 'True and False'. This might be easier to 'extend' in terms of types, but semantically its cofusing, and no better than a boolean.